### ISSUE TITLE
WSLUSC giving error and creating folder in $HOME

### ISSUE THREAD

**Issue by thunderbirdtr** [wslutilities/wslu#43]
**Background Information:**
 - Build: 18282.100
 - WSL: Wlinux
 - Version: 1.1.25.0

**Describe the bug**
wslusc go wild ! :) 

![image](https://user-images.githubusercontent.com/1688848/49006445-2d1d5000-f17a-11e8-8bdc-603ff0f3200c.png)

**To Reproduce**
wslusc code(or anything else)

I literally did nothing but completely go wild and creating "folders" but they're probably all "commands" and variables.

**Comment by thunderbirdtr** [comment]
> what is the output of your ` wslupath -d -H`?

/usr/bin/wslupath: line 105: /mnt/c/WINDOWS/system32/cmd.exe: cannot execute binary file Exec format error.

**Comment by patrick330602** [comment]
Interesting.... what's the version of wslu you are using? (check using `wslsys -v`), also what's the output of ` /mnt/c/WINDOWS/system32/cmd.exe /c "echo %appdata%" 2>&1 | tr -d "\r"`?

**Comment by thunderbirdtr** [comment]
I just noticed something, after I opened the laptop and type comment to this issue I re-typed "wslupath -d -H" and give me a proper result : "C:\\Users\\(username)" 

And of course ;
 └> wslusc code
[info] Create shortcut code.lnk successful.

I'm gonna try to do how can I cause this error. But I'm pretty sure I did change anything super "system wide" side. Because mostly I did "python development in "user" folder didn't even touch the anything super important but I don't know why is it working now.
